src/vma/lwip/opt.h

```diff
@@ -721,7 +721,7 @@
  * in seconds. (does not require sockets.c, and will affect tcp.c)
  */
 #ifndef LWIP_TCP_KEEPALIVE
-#define LWIP_TCP_KEEPALIVE              0
```

Comment:
why do we still want to keep those defines?
better to remove them at all, isn't it?

Reply:
@igor-ivanov ,do you agree?

Reply:
> @igor-ivanov ,do you agree?

Let do identical to LWIP_TCP_TIMESTAMPS.
Keep LWIP_TCP_KEEPALIVE.